docs/apache-airflow/start/docker-compose.yaml

```diff
@@ -146,13 +146,54 @@ services:
 
   airflow-init:
     <<: *airflow-common
-    command: version
+    command:
+      - bash
+      - -c
+      - |
+        if [[ -z "${AIRFLOW_UID}" ]]; then
```

Comment:
Should the `$` character be escaped?

Reply:
Nope. Here we are checking the docker env - it will be filled before the script is passed to docker.